test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -7,6 +7,10 @@ import '@ciscospark/internal-plugin-conversation';
 import {switchToMessage} from '../../lib/menu';
 import {clearEventLog, getEventLog} from '../../lib/events';
 import {constructHydraId} from '../../lib/hydra';
+const request = require(`superagent`);
```

Comment:
Let's use the `import` method here instead of the `require` to keep things consistent.